kplc

```diff
@@ -47,7 +48,12 @@ run_kompile() {
 }
 
 run_interpreter() {
-    notif_exec krun --directory "${backend_dir}" "${run_file}" "$@"
+    if ${result_only}; then
+        notif_exec krun --directory "${backend_dir}" "${run_file}" "$@" |\
+            tr '\n' ' ' | sed "s/.*<k>\(.*\)~> \. .*/\1/g"
+    else
+        notif_exec krun --directory "${backend_dir}" "${run_file}" "$@"
+    fi
```

Comment:
Would this work? https://unix.stackexchange.com/questions/38310/conditional-pipeline

Makes it so we don't have to repeat the actual command to run.

```suggestion
    notif_exec krun --directory "${backend_dir}" "${run_file}" "$@" \
        ( ${result_only} && ( tr '\n' ' ' | sed "s/.*<k>\(.*\)~> \. .*/\1/g" ) || cat - )
```

Reply:
I made the changes along with a few syntax-related changes and it works. Thanks for the suggestion!